packages/ui/src/app/pages/Proposals/EmailNotification.tsx

```diff
@@ -0,0 +1,382 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+import NotificationLogo from '@/app/assets/images/NotificationLogo.png'
+import NotificationMark from '@/app/assets/images/NotificationMark.png'
+
+export const EmailNotification = () => {
+  const [showModal, setShowModal] = useState(false)
+  const [showModal280, setShowModal280] = useState(false)
+
+  const openModal = () => {
+    setShowModal(true)
+  }
+  const openModal280 = () => {
+    setShowModal280(true)
+  }
+
+  return (
+    <div>
+      <ButtonContainer>
+        <Button onClick={openModal}>600px Notification</Button>
+        <Button onClick={openModal280}>280px Notification</Button>
+      </ButtonContainer>
+      {showModal && (
+        <ModalOverlay>
+          <ModalContainer>
+            <ModalHeader>
+              <Link>
+                <img src={NotificationLogo} alt="Mark" />
+              </Link>
+            </ModalHeader>
+            <ModalContent>
+              <ContentTitle>Hi {'{name}'},</ContentTitle>
+              <ContentBody>New proposal {'{proposal name}'} is created... there are 2 days left to vote...</ContentBody>
```

Comment:
According to the issue https://github.com/Joystream/pioneer/issues/4274
you must create six variations of notifications. Please create the text of the contentbody and button as variables.